configure

```diff
@@ -97,15 +98,19 @@ while test "x$1" != x; do
         echo "ENABLE_XAMARIN=1" >> $CONFIGURED_FILE
         shift
         ;;
-	--enable-bitcode-on-ios)
-		# Private option
-		echo "ENABLE_BITCODE_ON_IOS=1" >> $CONFIGURED_FILE
-		shift
-		;;
-	--disable-packaged-llvm)
-		echo "DISABLE_DOWNLOAD_LLVM=1" >> $CONFIGURED_FILE
-		shift
-		;;
+    --disable-xamarin)
+    echo "ENABLE_XAMARIN=0" >> $CONFIGURED_FILE
```

Comment:
sure it should not be `ENABLE_XAMARIN=` (no `0`) ?

Reply:
You are right.